packages/common/scss/leaders.scss

```diff
@@ -2,6 +2,15 @@
   &__logo {
     max-width: 100%;
   }
+  &__callout {
+    align-items: center;
+    font-size: 1.25rem;
+    text-align: left;
+    cursor: default;
```

Comment:
This is a div... any reason a `cursor` is set? this would be the default anyway.